.github/workflows/test.yml

```diff
@@ -49,3 +49,19 @@ jobs:
             echo "CryptoKit.o has been checked in. Please remove this from your PR."
             exit 1
           fi
+  check-headers:
+    strategy:
+      fail-fast: false
+      matrix:
+        header: [security]
+    runs-on: macos-15
```

Comment:
We should run this test on all supported macos versions to make sure things are compatible among them.